layouts/soarian/r4/clinical.html

```diff
@@ -0,0 +1,3 @@
+<h1 id="clinical">Clinical</h1>
+
+<p>The Clinical category of HL7<sup>®</sup> FHIR<sup>®</sup> resources includes Summary, Diagnostics, Medications, Care Provision, Request and Response.</p>
```

Comment:
Add a comma (.) after Request, so the end of the sentence is:  Request, and Response.
PRODOC-169058.

Reply:
File update.

Reply:
+1.